core/src/main/kotlin/org/evomaster/core/output/oracles/SchemaOracle.kt

```diff
@@ -103,8 +122,8 @@ class SchemaOracle : ImplementedOracle() {
 
         //this differs between kotlin and java
         when{
-            format.isJava() ->lines.add(".that($variableName, $json_ref.getMap(\"\").keySet().containsAll(Arrays.asList($referenceKeys)))")
-            format.isKotlin() -> lines.add(".that($variableName, $json_ref.getMap<Any, Any>(\"\").keys.containsAll(Arrays.asList($referenceKeys)))")
+            format.isJava() ->lines.add(".that($variableName, ($json_ref as LinkedHashMap<*,*>).keys.containsAll(Arrays.asList($referenceKeys)))")
+            format.isKotlin() -> lines.add(".that($variableName, ($json_ref as LinkedHashMap<*,*>).keys.containsAll(Arrays.asList($referenceKeys)))")
         }
```

Comment:
this seems brittle... ie, referifng to concrete class `LinkedHashMap` instead of the interface `Map`